### PR TITLE
feat: Add themes

### PR DESCRIPTION
**Pull request by AkshatGKMit**
- Create theme context
- Configure platform based status bar

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+import WrappedApp from '@screens/index';
 
 const App = () => {
   return (
-    <SafeAreaView>
-      <Text>Cinetube</Text>
-    </SafeAreaView>
+    <ThemeContextProvider>
+      <SafeAreaProvider>
+        <WrappedApp />
```
Comment (02priyanshi) [inline App.tsx]: use proper names

File: src/components/platformStatusBar/PlatformStatusBar.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { useContext } from 'react';
+import { View, StatusBar } from 'react-native';
+import ThemeContext from '@contexts/ThemeContext';
+import { isAndroid } from '@helpers/helpers';
```
Comment (02priyanshi) [inline src/components/platformStatusBar/PlatformStatusBar.tsx]: add this in constant file

File: src/constants/colors.ts [hunk 3]
```diff
@@ -0,0 +1,129 @@
+export const Light = {
+  c1: '#7a18e3',
+  c2: '#691abf',
+  c3: '#581a9d',
```
Comment (02priyanshi) [inline src/constants/colors.ts]: use proper names for better understanding

**Review by 02priyanshi — CHANGES_REQUESTED** [review]

**Comment by AkshatGKMit** [comment]
All naming issues are fixed

**Review by 02priyanshi — APPROVED** [review]

**Pull request merged**